### PR TITLE
Feat/login-page

### PR DESCRIPTION
**Pull request by alexsicart**

### CODE REVIEW

**Review by jmporchet — CHANGES_REQUESTED** [review]
not ready for merge, lacks CSS styling and redirection

**Review by alexsicart — COMMENTED** [review]
Ready for merge, but lacks of css in the sign up part, needs @arol reviews.

File: src/components/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class Homepage extends React.Component {
+
+  handleSubmit = (formData) => {
+    const {username, password} = formData;
+    this.props.login(username, password);
+  }
+
+  render() {
+    if (this.props.loggedIn) {
+      return <Redirect to="/preferences" />
```
Comment (arol) [inline src/components/Homepage.js:18]: Keep in mind that this only occurs the first time. I would redirect to `/` and then there (`Authenticated` component), if the user miss some config, you always kick it back to `/preferences`

**Review by arol — DISMISSED** [review]

**Comment by arol** [comment]
One does not simply get a conflict on the `package-log.json` file

**Review by jmporchet — APPROVED** [review]

**Pull request merged**